src/inventory_ui.cpp

```diff
@@ -885,7 +885,7 @@ static int num_parents( item_location loc )
     return 2 + num_parents( loc.parent_item() );
 }
 
-void inventory_column::draw( const catacurses::window &win, size_t x, size_t y ) const
+void inventory_column::draw( const catacurses::window &win, size_t x, size_t y )
```

Comment:
i know you said it would make the code more ugly, but i think returning drawn_info in this function might be better than making it non-const - a draw function should really be enforced as being const as drawing should be the only thing it does.